### PR TITLE
JAX-WS and JAX-RS 2.1: Prep CXF Core project source for CXF 3.5

### PR DESCRIPTION
**Pull request by neuwerk**
This PR does makes the following changes to the `com.ibm.ws.org.apache.cxf.cxf.core.3.2` project:

- Brings in all non-conflicting changes to the sources overlays from the 3.5.5 version of CXF
- Properly notates all liberty specific changes to the overlays
- Indicates changes that could be committed back to CXF 3.5 in future releases
- Indicates what overlays could be removed once the 3.5.5 update is complete.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBUtils.java [hunk 1]
```diff
@@ -639,35 +645,31 @@ public static Object setNamespaceMapper(Bus bus, final Map<String, String> nspre
...
         try {
             Class<?> cls;
             Class<?> refClass;
-            //Liberty change begin
+            //Liberty change begin: Check for Jakarta JAXB Context
             if (isEE9OrHigher) {
                 cls = Class.forName("org.glassfish.jaxb.runtime.api.JAXBRIContext");
                 refClass = Class.forName("org.glassfish.jaxb.runtime.api.TypeReference");
             } else {
+			   String pkg = "com.sun.xml.bind.";
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBUtils.java:665]: This seems un-needed and makes things more cluttered.

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/spi/NamespaceClassGenerator.java [hunk 2]
```diff
@@ -68,7 +68,7 @@ public synchronized Class<?> createNamespaceWrapperClass(Class<?> mcls, Map<Stri
             try {
                 byte[] bts = createNamespaceWrapperInternal(postFix);
                 className = "org.apache.cxf.jaxb.NamespaceMapper" + postFix; 
-                
+                // handle IBM namespace wrapper look up
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/spi/NamespaceClassGenerator.java:71]: // Liberty Change?

**Review by WhiteCat22 — APPROVED** [review]

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#libby

**Pull request merged**